src/composables/useLocalStorage.js

```diff
@@ -0,0 +1,24 @@
+import { ref } from 'vue'
+
+export default function useLocalStorage (key, initialValue) {
```

Comment:
Would be good to document this using a JSDoc comment in the format:
```suggestion
/**
 * Description here
 *
 * @param {string} key
 * @param {*} initialValue
 */
export default function useLocalStorage (key, initialValue) {
```
In VSCode if you type `/**` on the line above a function it should come up with an autocomplete suggestion called "JSDoc comment" which will automatically list the params for you